cmake/HPX_AddConfigTest.cmake

```diff
@@ -506,6 +506,15 @@ function(hpx_check_for_cxx20_perfect_pack_capture)
   )
 endfunction()
 
+# ##############################################################################
+function(hpx_check_for_nameable_threads)
+  add_hpx_config_test(
+    HPX_HAVE_NAMEABLE_THREADS
+    SOURCE cmake/tests/set_thread_name.cpp
+    FILE ${ARGN}
+  )
+endfunction()
```

Comment:
```suggestion
function(hpx_check_for_pthread_setname_np)
  add_hpx_config_test(
    HPX_WITH_PTHREAD_SETNAME_NP
    SOURCE cmake/tests/set_thread_name.cpp
    FILE ${ARGN}
  )
endfunction()
```

Reply:
Please also rename the feature test file itself reflecting that it checks for `pthread_setname_np`.